targets/chimera-host/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@
 # Viviane Potocnik <vivianep@iis.ee.ethz.ch>
 # Philip Wiese <wiesep@iis.ee.ethz.ch>
 
+
+set(TARGET "chimera-host")
```

Comment:
Why not reuse the already defined `TARGET_PLATFORM` variable? Please also adjust in the other files.

Reply:
I’ve added a comprehensive header `comment to stub_div64.c` explaining:

- Why OpenTitan provides trap-only builtins for 64-bit div/mod.
- How Picolibc on RV32 legitimately needs real 64-bit helpers.
- How we configure CMake to link the compiler-rt builtins so those calls resolve.
- Why we still need this stub to satisfy the linker and catch unintended uses on device.

Let me know if this clarifies your question. These changes are addressed in commit TBD.

Reply:
This variable is actually not used. I removed them everywhere in [7fdbd45](https://github.com/pulp-platform/chimera-sdk/pull/11/commits/7fdbd4544eb210556c28c11a1428732ba0b1384c).